mk/localdev.mk

```diff
@@ -9,6 +9,8 @@
 .PHONY : start-local
 start-local: check-venv generate_local_pg_tls_cert compose-up restart_django_foreground
 
+.PHONY : start-local-gunicorn
```

Comment:
Should we make this default ?

Reply:
I avoided it for two reasons:

* Didn't want to disrupt the status quo of local development
* Debugging code with multiple pods is harder

Eventually we could probably have it be the default with an nginx reverse-proxy in front and everything but for now this works